CRM/Event/Form/Registration/Confirm.php

```diff
@@ -733,10 +728,8 @@ public function postProcess() {
           }
         }
       }
-      if ($invoicing) {
-        $this->assign('dataArray', $dataArray);
-        $this->assign('totalTaxAmount', $totalTaxAmount);
-      }
+      $this->assign('dataArray', $dataArray);
```

Comment:
These will be NULL if invoicing is not enabled, better to assign in case of leakage